<?xml version="1.0"?>

<launch>
  <node pkg="hector_motion_detection" type="motion_detection" name="motion_detection" output="screen" respawn="true">
    <remap from="image_percept" to="worldmodel/image_percept" />
    <param name="image_transport" value="raw" />
  </node>

  <include file="$(find hector_motion_detection)/launch/load_robocup_params.launch" />
</launch>
